Magento_Checkout/web/template/minicart/item/default.html

```diff
@@ -1,99 +1,166 @@
+                                placeholder: qty,
+                                'data-cart-item': item_id,
+                                'data-item-qty': qty,
+                                'data-cart-item-id': product_sku
+                            },
+                            value: qty,
+                        "
+                        size="4"
+                        maxlength="12"
+                        autocomplete="off"
+                    >
+                </div>
+                <button
+                    data-bind="
+                        attr: {
+                            id: 'update-cart-item-'+item_id,
+                            'data-cart-item': item_id,
+                            title: $t('Update')
+                        }
+                    "
+                    class="
+                        button
+                        button--secondary
+                        minicart-product__update
+                        update-cart-item
+                    "
+                    style="display: none"
+                >
                     <span data-bind="i18n: 'Update'"></span>
                 </button>
             </div>
-            <div class="minicart__action">
+            <div class="minicart-product__actions">
                 <!-- ko if: is_visible_in_site_visibility -->
-                    <a data-bind="attr: {href: configure_url, title: $t('Edit item')}"
-                       class="action edit button button--icon">
-                        <svg class="button__icon">
-                            <use data-bind="attr: { 'xlink:href': require.toUrl('images/icons-sprite.svg#edit') }" xlink:href='' ></use>
+                    <a
+                        data-bind="attr: {href: configure_url, title: $t('Edit item')}"
+                        class="button button--icon minicart-product__button"
+                        aria-label="edit product in your shopping cart"
+                    >
+                        <svg
+                            title="edit"
+                            class="button__icon minicart-product__button-icon"
+                        >
+                            <use data-bind="attr: { 'xlink:href': require.toUrl('images/icons-sprite.svg#edit') }" xlink:href=''></use>
                         </svg>
                     </a>
                 <!-- /ko -->
-                    <button data-bind="attr: {'data-cart-item': item_id, title: $t('Remove item')}"
-                       class="action delete button button--icon"
+
+                <a
+                    href="#"
+                    data-bind="attr: {'data-cart-item': item_id, title: $t('Remove item')}"
+                    class="button button--icon minicart-product__button action delete"
+                    aria-label="remove this product from your shopping cart"
+                >
+                    <svg
+                        title="remove"
```

Comment:
we don't need title attribute in svg tag

Reply:
Removed